packages/babel-make-styles/__fixtures__/error-style-method/fixture.js

```diff
@@ -0,0 +1,6 @@
+import { makeStyles } from '@fluentui/react-make-styles';
+
+// This file is .js intentionally to avoid TS compiler errors
+export const useStyles = makeStyles({
+  root() {},
```

Comment:
what kinds of errors do you get from this test case ?

Reply:
This throw in this part of code:

```ts
        propertiesPaths.forEach(propertyPath => {
          if (propertyPath.isObjectMethod()) {
            throw propertyPath.buildCodeFrameError('Object methods are not supported for defining styles');
          }
```

You can find matching error message in `packages/babel-make-styles/src/plugin.test.ts`. For customers it will look like this:

![image](https://user-images.githubusercontent.com/14183168/115435064-c052f680-a209-11eb-9262-5cbb7d60735e.png)